content/en/docs/community/release_checklist.md

```diff
@@ -165,12 +200,14 @@ need to update corresponding tests that are using that version number.
 
 ```shell
 git add .
-git commit -m "bump version to $RELEASE_CANDIDATE_NAME"
+git commit -m "bump version to $RELEASE_NAME"
```

Comment:
I have a note from my shadowing session with @bacongobbler that this was wrong in the doc.